src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -401,10 +408,13 @@ abstract class ManagedStrategy(
     }
 
     override fun onActorStart(iThread: Int) {
-        currentActorId[iThread]++
-        callStackTrace[iThread].clear()
-        suspendedFunctionsStack[iThread].clear()
-        loopDetector.onActorStart(iThread)
+        runInIgnoredSection {
```

Comment:
Please explain why it is essential to run this code in an ignored section. Maybe, it's not.